compiler/circle-part-value-test/parts/SignatureDef_MultiOut_000.part

```diff
@@ -0,0 +1,7 @@
+[partition]
+backends=cpu,acl_cl
+default=cpu
+comply=opcode
+
+[OPCODE]
+MAXIMUM=acl_cl
```

Comment:
`MAXIMUM` is just random OP and doesn't matter cause this test is to validate `part_eval_one.py` is working OK with SignatureDef